rllib/tests/test_external_multi_agent_env.py

```diff
@@ -64,32 +58,6 @@ def test_external_multi_agent_env_sample(self):
         batch = ev.sample()
         self.assertEqual(batch.count, 50)
 
-    def test_train_external_multi_agent_cartpole_many_policies(self):
-        n = 20
```

Comment:
Do we have a replacement for this test?

Reply:
I'm not sure this was that useful a test (it's just evaluating with 20 policies, and isn't actually testing external envs), so just decided to drop it.

Reply:
Ok.